src/MobSpawner.cpp

```diff
@@ -236,6 +236,32 @@ bool cMobSpawner::CanSpawnHere(cChunk * a_Chunk, int a_RelX, int a_RelY, int a_R
 				}
 				return CanSpawn && HasFloor && (SkyLight <= 7) && (BlockLight <= 7);
 			}
+
+			case mtCaveSpider:
+			{
+				bool CanSpawn = true;
+				bool HasFloor = false;
+				for (int x = 0; x < 1; ++x)
+				{
+					for (int z = 0; z < 1; ++z)
+					{
```

Comment:
These two loops are not needed at all, they basically just say 'int x = 0;` and `int z = 0`, since they are never repeated.